errors/errors.go

```diff
@@ -174,4 +174,5 @@ var (
 	ErrInvalidSearchQuery             = errors.New("invalid search query")
 	ErrImageNotFound                  = errors.New("image not found")
 	ErrAmbiguousInput                 = errors.New("input is not specific enough")
+	ErrUnexpectedAuthHeader           = errors.New("expected bearer auth header, received basic")
```

Comment:
The string does not match the variable name exactly. How about having a more generic string here, and wrapping the error adding the specifics of 'bearer'/basic' in the code where it is used?

Reply:
@eusebiu-constantin-petu-dbk ^?

Reply:
Not sure we need this PR if we merge https://github.com/project-zot/zot/pull/2903

Reply:
is it better?

Reply:
yes